build.sbt

```diff
@@ -4,9 +4,10 @@ lazy val root = (project in file("."))
     organization := "github.discordscala",
     scalaVersion := "2.12.4",
     version      := "0.1.0-SNAPSHOT",
-    libraryDependencies ++= Seq("net.liftweb" %% "lift-json" % "3.1.1",
+    libraryDependencies ++= Seq(
+      "net.liftweb" %% "lift-json" % "3.1.1",
```

Comment:
Here formatting seems a little bit messed up relative to the other entries

Reply:
You usually
```
Seq(
  element,
  element,
)
```
in a multiline seq